src/workspaces/cookiefactoryv2/README.md

```diff
@@ -56,21 +104,59 @@ This project walks you through the process of setting up the Bakersville Cookie
       ```
     - Open scene in console: `https://us-east-1.console.aws.amazon.com/iottwinmaker/home?region=us-east-1#/workspaces/__FILL_IN__/scenes/CookieFactory`
 
-4. Setup Web Application
-   - Follow instructions in [CookieFactoryDemo](./CookieFactoryDemo/README.md)
+
+
+## Configure Web Application
+
+First change directory to CookieFactoryDemo
+``` 
+cd ../CookieFactoryDemo 
+```
+
+1. Change `WORKSPACE_ID` in `src/config/sites.ts` to your workspace ID.
+2. Use the following CLI command to administratively set the password for your demo Cognito user, Fran. Your User_Pool_id can be found after your CDK deploy completed in the CloudFormationOutput (printed to console). The demo Username is "fran@cookiefactory". Be sure to create a password that meets the default Cognito password requirements (Lowercase letter, Uppercase letter, Number, Symbol, Length >= 8)
+```
+aws cognito-idp admin-set-user-password --user-pool-id "[YOUR_USER_POOL_ID]" --username "[USERNAME]" --password "[PASSWORD]" --permanent
+```
+
+3. Set your AWS and Cognito user credentials in: `src/config/cognito.template.ts` and `src/config/user.template.ts` and rename the files to `src/config/cognito.ts` and `src/config/users.ts`, respectively. You will need your newly created Cognito UserPoolId, IdentityPoolId, ClientId, Username and password for your user
+
+OPTIONAL: view [Console instructions](./CookieFactoryDemo/COGNITO_SAMPLE_SETUP_CONSOLE.md) for sample Amazon Cognito configuration (already created through CDK). You may use Cognito to configure custom security settings for your User Pools.
+
+### Install
+
+```shell
+npm install
+```
+
+### Development server
+
+* Edit `webpack.dev.js` if needed to change the port for the application (default is port 5000)
+
+```shell
+npm run dev
+```
+
+* Navigate to the application at `localhost:5000` (default) - Note: it may take a minute to load the first time
+
+### Build
+
+```shell
+npm run build
+```
+---
 
 ## Cleanup
 
 1. Delete resources using CDK (note: can also be done in AWS Console / CLI against the CloudFormation stack)
     - cdk destroy
         ```
         cdk destroy \
-          --context stackName="__FILL_IN__" \
-          --context iottwinmakerWorkspaceId="__FILL_IN__" \
-          --context iottwinmakerWorkspaceBucket="__FILL_IN__"
+          --context stackName="CookieFactoryDemo" \
+          --context iottwinmakerWorkspaceId="$WORKSPACE_ID" \
+          --context iottwinmakerWorkspaceBucket="$WORKSPACE_BUCKET_NAME"
         ```
-2. Delete any Cognito-related resources setup for the demo if needed
-
+2. Navigate to the AWS Cognito console and delete CookiefactoryUserPool in the "User pools" tab by selecting the name, clicking "delete", and confirming the name in the prompt.
```

Comment:
can we provide the CLI command(s) in interim instead of having them go to console?

Reply:
Yes I'll change to cli for now. Ideally the CDK should destroy it, I still am unsure why only the user pool doesn't get deleted by CDK

Reply:
wonder if could be related to termination protection: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cognito-userpool.html#cfn-cognito-userpool-deletionprotection

Reply:
That was my first guess too since deletion protection is enabled in our original instructions, but I explicitly disabled it in the cdk code here
line 403: `deletionProtection: false,`